tests/unit/initializers/app-initializer-test.js

```diff
@@ -0,0 +1,26 @@
+import Application from '@ember/application';
+import { run } from '@ember/runloop';
+
+import { initialize } from 'ember-twiddle/initializers/app-initializer';
+import { module, test } from 'qunit';
+import destroyApp from '../../helpers/destroy-app';
+
+module('Unit | Initializer | app initializer', {
+  beforeEach() {
+    run(() => {
+      this.application = Application.create();
+      this.application.deferReadiness();
+    });
+  },
+  afterEach() {
+    destroyApp(this.application);
```

Comment:
This helper is for old "moduleForAcceptance" tests. I think you can remove it.

Reply:
nvm. It's in my code. You just have to edit the line in `tests/helpers/destroy-app` from `window.server.shutdown()` to `window.server && window.server.shutdown();`